src/query.c

```diff
@@ -620,7 +651,7 @@ static void rangeIterCbStrs(const char *r, size_t n, void *p, void *invidx) {
   rangeItersAddIterator(ctx, ir);
 }
 
-static int rangeIterCb(const rune *r, size_t n, void *p) {
+static int runeIterCb(const rune *r, size_t n, void *p, void *payload) {
```

Comment:
Where do you use the payload? Can you explain what the payload is?

Reply:
expanded top comment with info.